<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gBox_hisSlide">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Histogram Slide</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_hisSlide">
      <item>
       <widget class="QPushButton" name="btn_hisSlide_gray">
        <property name="text">
         <string>Apply (Gray Image)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_hisSlide_Color">
        <property name="text">
         <string>Apply (Color Image)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_label">
     <item>
      <widget class="QLabel" name="lb_fileName">
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_c">
       <property name="text">
        <string>By the amount</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>771</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_lineEdit">
     <item>
      <widget class="QLineEdit" name="ln_fileName"/>
     </item>
     <item>
      <widget class="QLineEdit" name="ln_c">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_browse">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gBox_hisStretch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Histogram Stretch</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_hisStretch">
      <item>
       <widget class="QPushButton" name="btn_hisStretch_gray">
        <property name="text">
         <string>Apply (Gray Image)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_hisStretch_Color">
        <property name="text">
         <string>Apply (Color Image)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBox_segmentation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Segmentation</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_segmentation_gray">
        <property name="text">
         <string>Apply (Gray Image)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_segmentation_Color">
        <property name="text">
         <string>Apply (Color Image)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBox_showHistogram">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Show Histogram</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_showHis">
      <item>
       <widget class="QPushButton" name="btn_showHis_gray">
        <property name="text">
         <string>Apply (Gray Image)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_showHis_Color">
        <property name="text">
         <string>Apply (Color Image)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBox_linearModification">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Linear Modification</string>
    </property>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_linearModification">
      <item>
       <widget class="QPushButton" name="btn_linearModification_gray">
        <property name="text">
         <string>Apply (Gray Image)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_linearModification_Color">
        <property name="text">
         <string>Apply (Color Image)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBox_histogramEqual">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Histogram Equalization</string>
    </property>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_histogramEqual">
      <item>
       <widget class="QPushButton" name="btn_histogramEqual_gray">
        <property name="text">
         <string>Apply (Gray Image)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_histogramEqual_Color">
        <property name="text">
         <string>Apply (Color Image)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>btn_browse</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>gBox_hisSlide</zorder>
   <zorder>gBox_hisStretch</zorder>
   <zorder>gBox_segmentation</zorder>
   <zorder>gBox_showHistogram</zorder>
   <zorder>gBox_linearModification</zorder>
   <zorder>gBox_histogramEqual</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
